### PR TITLE
Add FAQ about block filters

### PR DESCRIPTION
**Pull request by RiccardoMasutti**

### CODE REVIEW

File: docs/FAQ/FAQ-Introduction.md [hunk 1]
```diff
@@ -125,6 +125,14 @@ Furthermore our calculations have shown that with the liquidity of today’s mix
 Yes, you can check the status of Wasabi-related services and websites (like APIs, Backend, etc.) via [UptimeRobot Wasabi Status Page](https://stats.uptimerobot.com/YQqGyUL8A7).
 :::
 
+:::details
+### What software supplies the block filters that Wasabi uses?
+
+The zkSnacks coordinator supplies the filters.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Introduction.md]:
```suggestion
The zkSNACKs coordinator supplies the filters.
```

**Comment by RiccardoMasutti** [comment]
@MaxHillebrand @yahiheb Ok for me

File: docs/FAQ/FAQ-Introduction.md [hunk 2]
```diff
@@ -125,6 +125,14 @@ Furthermore our calculations have shown that with the liquidity of today’s mix
 Yes, you can check the status of Wasabi-related services and websites (like APIs, Backend, etc.) via [UptimeRobot Wasabi Status Page](https://stats.uptimerobot.com/YQqGyUL8A7).
 :::
 
+:::details
+### What software supplies the block filters that Wasabi uses?
+
+The zkSNACKs coordinator supplies the filters.
+This means you rely on the server providing you valid filters.
+But because you download the blocks from a random Bitcoin peer-to-peer node - or your own node - the coordinator cannot spy on which blocks you are interested in.
+:::
```
Comment (molnard) [inline docs/FAQ/FAQ-Introduction.md]: which blocks you are interested in. Furthermore, the random node will only know which block is needed but it won't have any clue which transaction(s) belongs to the wallet. (Can be rephrased if you know simpler form)

**Comment by RiccardoMasutti** [comment]
@molnard @MaxHillebrand ready

**Pull request merged**